internal/ir/xds.go

```diff
@@ -131,6 +131,10 @@ type HTTPRoute struct {
 	QueryParamMatches []*StringMatch
 	// Destinations associated with this matched route.
 	Destinations []*RouteDestination
+	// Redirections to be returned for this route. Takes precedence over Destinations
```

Comment:
can you update the order of the fields to reflect the priority mentioned in the comment